aws/vpc-peering/main.tf

```diff
@@ -0,0 +1,80 @@
+terraform {
+  required_version = ">= 0.11.2"
+
+  backend "s3" {}
+}
+
+provider "aws" {
+  assume_role {
+    role_arn = "${var.aws_assume_role_arn}"
+  }
+}
+
+# Fetch the OrganizationAccountAccessRole ARNs from SSM
+module "requester_role_arns" {
+  enabled        = "${var.enabled}"
+  source         = "git::https://github.com/cloudposse/terraform-aws-ssm-parameter-store?ref=tags/0.1.5"
+  parameter_read = ["/${var.namespace}/${var.requester_account}/organization_account_access_role"]
+}
+
+locals {
+  requester_vpc_tags = "${var.requester_vpc_tags}"
+  requester_region   = "${var.requester_region}"
+  requester_role_arn = "${join("", module.requester_role_arns.values)}"
+}
+
+# Fetch the OrganizationAccountAccessRole ARNs from SSM
+module "accepter_role_arns" {
+  enabled        = "${var.enabled}"
+  source         = "git::https://github.com/cloudposse/terraform-aws-ssm-parameter-store?ref=tags/0.1.5"
+  parameter_read = ["/${var.namespace}/${var.accepter_account}/organization_account_access_role"]
+}
+
+locals {
+  accepter_vpc_tags = "${var.accepter_vpc_tags}"
+  accepter_region   = "${var.accepter_region}"
```

Comment:
looks like these two locals are redundant

Reply:
Just for symmetry ....